go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/MarinX/monerorpc
 
-go 1.14
+go 1.19
```

Comment:
Hopefully, there is no one still developing new code with 1.14. I updated to the latest version that is still getting security and other bug fixes and did a `go get -u ./... && go mod tidy`.